doc/source/changes/version_0_27.rst.inc

```diff
@@ -171,3 +174,5 @@ Fixes
 -----
 
 * fixed array values being editable in view() (instead of only in edit()).
+
+* fixed `sort_values` failing when passing only `ascending` argument (closes :issue:`542`).
```

Comment:
remove from changelog as it was not present in 0.26.1 (could not call sort_values without key).